### PR TITLE
Ana/fix polkadot calculate network fees

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Small bug. Killed it

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Still, ClaimRewardsTx is trhowing this error:

```
2020-06-21 12:53:13        RPC-CORE: queryInfo(extrinsic: Bytes, at?: BlockHash): RuntimeDispatchInfo:: 2: Unable to query dispatch info.: Error decoding field Call :: Utility.0
```

:eyes:

**Comment by Bitcoinera** [comment]
I also fixed this view:

**Comment by faboweb** [comment]
🚀 Deployed on https://5eef3e7ae9fafeace19062e0--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
OK, I just added a back up plan that seems to be working for Kusama at least. Not a real fix but still something that I believe we should have there (specially taking into account the constant upgrades).

Still, getting rn for Kusama in ClaimRewards this afterwards, but this looks like another issue altogether:

File: api/data/network-fees.js [hunk 1]
```diff
@@ -204,35 +204,44 @@ const getPolkadotMessage = async (messageType, senderAddress, message, network,
...
+    if (message.amounts) {
+      const { amounts } = message
+      amount = amounts[0]
+    }  
+    return {
+      denom: (amount && amount.denom) || network.stakingDenom,
+      amount: viewFees
+    }
+  } catch(error) {
+    console.error(error)
```
Comment (faboweb) [inline api/data/network-fees.js:238]: let's catch the error in Sentry?

Comment (Bitcoinera) [reply api/data/network-fees.js:238]: ok!

**Comment by Bitcoinera** [comment]
I was going to do it :sweat_smile:

**Pull request merged**